### PR TITLE
Upgrade salt to 3002.5

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
**Component**:

'salt'

**Context**: 

Some CVE was discovered in Salt 3002.2 https://saltproject.io/security_announcements/active-saltstack-cve-release-2021-feb-25/

**Summary**:

- Bump Salt version to 3002.2
- Add `roster` in Salt master configuration file (sees https://github.com/saltstack/salt/issues/59657)

---

### CODE REVIEW

File: images/salt-master/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM centos:7.6.1810
 MAINTAINER moonshot-platform <moonshot-platform@scality.com>
 
 # Versions to use
-ARG SALT_VERSION=3002.2
+ARG SALT_VERSION=3002.5
```
Comment (alexandre-allard) [inline images/salt-master/Dockerfile]: The Salt version is already passed to the container build context from the buildchain, it should be:
```suggestion
ARG SALT_VERSION
```
This way we don't have to maintain the version in 2 places.

Comment (TeddyAndrieux) [reply images/salt-master/Dockerfile]: Right, it's not linked to this PR, but I agree

Comment (TeddyAndrieux) [reply images/salt-master/Dockerfile]: Done

**Review by alexandre-allard — APPROVED** [review]

**Pull request merged**